runelite-client/src/test/java/net/runelite/client/plugins/combatlevel/CombatLevelPluginTest.java

```diff
@@ -262,8 +253,27 @@ public void testPlayerZezima()
 			player.getCombatLevel() + 1, DEF_HP_MULT));
 
 		// test prayer
-		int prayNeed = calcLevels(baseValues.get("base") + baseValues.get("max"),
-			player.getCombatLevel() + 1, PRAY_MULT);
-		assertEquals(4, correctPrayer(client.getRealSkillLevel(Skill.PRAYER), prayNeed));
+		assertEquals(4, calcLevelsPray(baseValues.get("base") + baseValues.get("max"),
+			player.getCombatLevel() + 1, client.getRealSkillLevel(Skill.PRAYER)));
+	}
+
+	@Test
+	public void testPrayerLevelsNeeded()
+	{
+		// Check an edge case where the needed prayer levels to next combat could be calculated to be 3 instead of 1
```

Comment:
How is this an "edge case"?

Reply:
Good point, I'll move this to the commit message.